nixos/modules/programs/ssh.nix

```diff
@@ -203,6 +203,9 @@ in
     # generation in the sshd service.
     environment.etc."ssh/ssh_config".text =
       ''
+        ${cfg.extraConfig}
+
+        Host *
```

Comment:
What's this for?

Reply:
The `Host *` is so the following config options apply to all hosts if they haven't been set before. For example if `cfg.extraConfig` is:
```
Host example.com
    Port 10022
```

then the options below would only apply to `example.com` without the `Host *` - ssh options are not indentation sensitive, that is just for readability.